src/markdown-pages/docs/managers/zos-program-manager.md

```diff
@@ -142,15 +106,89 @@ The following are properties used to configure the zOS Program Manager.
 </details>
  
 <details>
-<summary>zOS Program Language link SYSLIBs</summary>
+<summary>z/OS Program Language link SYSLIBs</summary>
 
 | Property: | zOS Program Language link SYSLIBs |
 | --------------------------------------- | :------------------------------------- |
 | Name: | zosprogram.[language].[imageid].link.syslibs |
-| Description: | The site specific and language specific (COBOL, C, PL1, ASSEMBLER) custom zOS data sets that contain load modules that are used in the link SYSLIB concatenation in the zOS program compile and link JCL. |
+| Description: | The site specific and language specific (COBOL, C, PL1, ASSEMBLER) custom z/OS data sets that contain load modules that are used in the link SYSLIB concatenation in the z/OS program compile and link JCL. |
 | Required:  | No |
 | Default value: | None |
 | Valid values: | A comma separated list of zOS data sets |
 | Examples: | <code>zosprogram.cobol.MVSA.link.syslibs=TEAM.LOADLIB</code><br> <code>zosprogram.cobol.link.syslibs=COMPANY.LOADLIB,TEAM.LOADLIB</code> |
 
 </details>
+
+
+# <a name="annotations"></a>Annotations provided by the Manager
+
+The following annotations are available with the z/OS Program Manager
+<details>
+<summary>z/OS Program</summary>
+
+| Annotation: | z/OS Program |
+| --------------------------------------- | :------------------------------------- |
+| Name: | @ZosProgram |
+| Description: | The <code>@ZosProgram</code> annotation requests the z/OS Program Manager to Compile and Bind a program on a z/OS image.  The test can request multiple z/OS Program instances |
+| Attribute: `name` |  The program name |
+| Attribute: `location` |  Path to the location of the program source in the Galasa test bundle. This can be either the full path including the file name or the directory containing the source with the name specified in the name attribute with the extension specified in the language attribute.  |
+| Attribute: `language` |  The programming language. See <a href="https://javadoc.galasa.dev/dev/galasa/zosprogram/ZosProgram.Language.html" target="_blank" rel="noopener noreferrer">ZosProgram.Language</a>. <br><br>  |
+| Attribute: `cics` |  Is a CICS program and requires the CICS translator. |
+| Attribute: `loadlib` |  The load module data set name |
+| Attribute: `imageTag` |  The <code>imageTag</code> is used to identify the z/OS image. |
+| Attribute: `compile` |  Compile this zOS program. |
+| Syntax: | @ZosImage(imageTag="A")<br> public IZosImage zosImageA;<br> @ZosProgram(imageTag="A")<br> public IZosProgram zosProgramA;<br></code> |
+| Notes: | The <code>IZosProgram</code> interface has a number of methods to manage the z/OS Program. See <a href="https://javadoc.galasa.dev/dev/galasa/zosprogram/ZosProgram.html" target="_blank">ZosProgram</a> and <a href="https://javadoc.galasa.dev/dev/galasa/zosprogram/IZosProgram.html" target="_blank">IZosProgram</a> to find out more. |
```

Comment:
We should list if these are optional or not and if they have a default value

Reply:
Added